### PR TITLE
docs: translate zh/plugins/files

### PR DESCRIPTION
**Pull request by WingLim**
- [x] files.md

### CODE REVIEW

File: site/docs/plugins/files.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# File Handling Simplified in grammY
+# File Handling Simplified in grammY (`files`)
```
Comment (KnorpelSenf) [inline site/docs/plugins/files.md:1]: Nice!

**Review by KnorpelSenf — COMMENTED** [review]
I'm going to have to disable deploy previews now. Automatic deployments to the live version will keep on working.

The previews will be enabled again after November 15.

**Comment by WingLim** [comment]
> I'm going to have to disable deploy previews now. Automatic deployments to the live version will keep on working.
> 
> The previews will be enabled again after November 15.

Good to know, there are not many Chinese documents left to be translated.

**Comment by KnorpelSenf** [comment]
You did an excellent job! 🎉

**Review by linbuxiao — APPROVED** [review]
帅！

**Pull request merged**